Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -512,8 +512,9 @@ ExplicitBitVect *GetMorganFingerprintBV(
   }
   ExplicitBitVect *res;
   res = RDKit::MorganFingerprints::getFingerprintAsBitVect(
-      mol, static_cast<unsigned int>(radius), nBits, invars, froms.get(),
-      useChirality, useBondTypes, false, bitInfoMap,
+      mol, static_cast<unsigned int>(radius), static_cast<unsigned int>(nBits),
```

Comment:
since radius and nBits are both unsigned now, you shouldn't need these casts at all:
```suggestion
      mol, radius, nBits,
```